mesheryctl/internal/cli/root/registry/generate.go

```diff
@@ -199,8 +206,10 @@ func InvokeGenerationFromSheet(wg *sync.WaitGroup) error {
 				return
 			}
 
-			if model.Registrant == "artifacthub" {
-				time.Sleep(10 * time.Second)
+			if model.Registrant == "Artifact Hub" {
```

Comment:
@Jougan-0 in the model.go file just format the registrant name using `ReplaceSpacesAndConvertToLowercase` (meshkit func) and then try if still ratelimitting is hit. (keeping the old logic for ratelimitting, as I think 5 mins is too long even if you are doing in bunch)

Reply:
Previously the registrant name was formatted as sheet was parsed through, but later I moved it so that the real name as listed in sheet gets utilised but missed update here. And before this I wasn't seeing 429s in the action run. 
I was of the belief that the sleep logic is being used but after seeing your PR I realised this if block was getting skipped because of the above mentioned change.

Reply:
Changing back to old logic and running the [action](https://github.com/Jougan-0/meshery/actions/runs/9447298507)  we are still getting the 429 error .
I don;t think so that 5 mins time is much because for every 100 artifacthub registrant we wait for 5 minutes so it takes 30 minutes at max now to run the whole script.
[Here](https://github.com/Jougan-0/meshery/actions/runs/9439977307) have a look.